File: backend/src/main/java/com/campus/Campus_Connect/features/map/dto/response/EventMarkerResponse.java
package com.campus.Campus_Connect.features.map.dto.response;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class EventMarkerResponse {

    private Integer id;

    private String title;

    private Double latitude;

    private Double longitude;

    private Integer priority;
}

File: android-app/app/src/main/java/com/example/campusconnect/feature/map/data/remote/response/EventMarkerResponse.kt
package com.example.campusconnect.feature.map.data.remote.response

import com.google.gson.annotations.SerializedName

data class EventMarkerResponse(


    val id: Int,


    val title: String,


    val latitude: Double? = null,


    val longitude: Double? = null,


    val priority: Int? = null
)
